src/test/resources/reference.conf

```diff
@@ -108,3 +108,8 @@ spray.can.host-connector {
 notification {
   fullyQualifiedNotificationTopic = "dummy"
 }
+
+staticNotebooks {
+  # OK to call deployed GAE dev for unit tests?
```

Comment:
Do we have tests that actually use this? Or do you just need to give it some non-empty value?

Reply:
This was not actually used - removed